katalog/fluentd/deploy.yml

```diff
@@ -19,7 +19,7 @@ spec:
         effect: NoSchedule
       containers:
       - name: fluentd
-        image: sighup/fluentd-elasticsearch:1.2.8
+        image: sighup/fluentd-elasticsearch:v1.7.2-debian-1.0
```

Comment:
The image does not exist. The right one is: `quay.io/sighup/fluentd-elasticsearch:v1.7.2-debian-1.0`.

Reply:
Yes, I'll fix this.